### PR TITLE
DSDEEPB-716 Exit cleanly with bad WDL.

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: src/test/scala/cromwell/util/SampleWdl.scala [hunk 1]
```diff
@@ -82,6 +82,38 @@ object SampleWdl {
...
+        |}
+        |
+        |
+        |workflow test1 {
+        |  String bfile
+        |  call summary {
+        |     input: bfile=bfile
+        |  }
+        |}
+      """.stripMargin
```
Comment (scottfrazer) [inline src/test/scala/cromwell/util/SampleWdl.scala]: So if I'm understanding this correctly, this WDL is supposed to be malformed?  If so, then I don't see how

Comment (mcovarr) [inline src/test/scala/cromwell/util/SampleWdl.scala]: Cromwell certainly doesn't like it as it is now.  :smile: Do we currently support workflow inputs this way?

Comment (scottfrazer) [inline src/test/scala/cromwell/util/SampleWdl.scala]: yeah, this will soon be valid unless I'm missing something obvious.  Does any syntax error work?

Comment (scottfrazer) [inline src/test/scala/cromwell/util/SampleWdl.scala]: in-person conversation:  I'll fix this in my branch when this syntax becomes valid.

**Comment by geoffjentry** [comment]
:+1:

**Pull request merged**